apps/web/src/app/api/checker/regions/arn1/route.ts

```diff
@@ -0,0 +1,9 @@
+import { checker } from "../_checker";
+
+export const runtime = "edge";
+export const preferredRegion = ["arn1"];
+export const dynamic = "force-dynamic";
+
+export async function POST(request: Request) {
```

Comment:
We could move it to a `GET` method and allow the `url` to be a search param.

Reply:
Somehow, I think `POST` is cleaner. Having the props inside the `request.body` is cleaner.

Reply:
Qstash post a Json to an endpoint :)